cranelift-codegen/meta/src/cdsl/formats.rs

```diff
@@ -75,6 +72,18 @@ impl InstructionFormat {
                 )
             })
     }
+
+    /// Returns a tuple that uniquely identifies the structure.
+    pub fn structure(&self) -> (usize, bool, Vec<&'static str>) {
```

Comment:
Ok, I see now how this is used; for clarity could this be a struct instead of an anonymous tuple? (if it introduces weird import issues then never mind)

Reply:
Sure!